### DISCUSSION TITLE
Weighted mode with most frequent 1000 words chosen by relative frequency

### DISCUSSION THREAD

**Discussion by SongWithoutWords** [monkeytypegame/monkeytype#1272] [Ideas]
Would people be interested in a mode where the most frequent thousand words in a language were chosen according to their relative frequency in the language (rather than a uniform frequency), so that the typing test was more representative of typing in that language?

If people are interested in this feature I could take a stab at implementing it.

**Comment by unrealapex** [comment]
I believe the word list for each language are not the same.

**Reply by SongWithoutWords** [reply]
Definitely wasn't meaning to imply that all languages share the same most frequent thousand words. This suggestion is to have a mode where words are chosen in accordance with their frequency within their language. Wiki page about word frequency here: https://en.m.wikipedia.org/wiki/Word_lists_by_frequency#:~:text=In%20computational%20linguistics%2C%20a%20frequency,the%20position%20in%20the%20list.

**Reply by unrealapex** [reply]
It seems interesting, try making a PR and see what @Miodec thinks about it.
